hera_pspec/uvpspec_utils.py

```diff
@@ -241,6 +242,10 @@ def _blpair_to_antnums(blpair):
     ant2 = int(np.floor(blpair / 1e6 - ant1*1e3))
     ant3 = int(np.floor(blpair / 1e3 - ant1*1e6 - ant2*1e3))
     ant4 = int(np.floor(blpair - ant1*1e9 - ant2*1e6 - ant3*1e3))
+    ant1 -= 100
```

Comment:
Why are these on separate lines? They could just be put into the lines above.

Reply:
I don't think so, see how ant2 is dependent on ant1 and so on..

Reply:
Oops, my bad.